src/plugins/no_data_page/README.md

```diff
@@ -0,0 +1,3 @@
+# No Data Page
+
+Helps to globally configure the no data page components
```

Comment:
Need this dummy plugin to create a config for configuring the analytics empty pages "flavor"

Reply:
An alternative could be that all discover, visualize, dashboards expose this config on their own and pass it to the component.